src/OpenLoco/src/Windows/Construction/Construction.h

```diff
@@ -13,9 +13,20 @@ using namespace OpenLoco::World::TileManager;
 
 namespace OpenLoco::Ui::Windows::Construction
 {
+    enum class GhostFlags : uint8_t
```

Comment:
i feel i'm repeating myself too much with these. Would `GhostVisibleFlags::arrow` make more sense?

Reply:
I agree that sounds better. Though perhaps `GhostVisibilityFlags`?